assets/css/custom.css

```diff
@@ -110,24 +112,110 @@ body, #qcSiteNav h4, #qcPriNav ul > li > a, #qcPriNav ul ul > li > a, .dash_titl
   width: 80%;
 }
 
+#qcCountdown {
+	color: #EB6851;
+	z-index: 11;
+}
+
+/* adapt page intro for mobile sticky nav  */
+@media only screen and (max-width: 767px) {
+	#slideContent, .page-intro {
+		height: 100%;
+	}
+	#qcHeader {
+		display: none;
+	}
+	#qcSlideText {
+		margin-top: 0;
+		padding-top: 30%;
+		height: 100%;
+	}
+	.cta .video .register-cta {
+		display: none;
+	}
+}
+
 @media only screen and (max-width: 335px) {
 	#qcSlideText h1 {
-	  font-size: 28px;
+	  font-size: 28px; /* shrink headline to fit on extra tiny screens */
+	}
+}
+
+/* INTRO REGISTER NOW BUTTON */
+
+#qcSlideText .cta {
+	color: #fff;
+	line-height: 60px;
+	background: #EB6851;
+	text-transform: uppercase;
+	border-radius: 30px;
+}
+#qcSlideText .cta:hover {
+	background: rgba(0,0,0,0.3);
+}
+@media only screen and (max-width: 767px) {
+	#qcSlideText .cta.video {
+		display: none;
+	}
+}
+
+/* MOBILE STICKY BAR */
+
+@media only screen and (min-width: 768px) {
+	#mobileNav {
+		display: none;
+	}
+}
+
+#qcPriNav #nav-register a {
+	color: #EB6851;
+}
+
+@media only screen and (max-width: 767px) {
+	#mobileNav {
+		background-color: #172751;
+		position: fixed;
+	  bottom: 0;
+	  left: 0;
+	  z-index: 990;
+	  width: 100%;
+	  height: 64px;
+	}
+	#qcOpenNav {
+	  background: url(../img/theme/openNavInvert.png) center center;
+	  top: 6px;
+	}
+
+	/* MOBILE STICKY REGISTER NOW BUTTON */
+
+	.cta {
+		padding: 10px 20px;
+		color: #fff;
+		line-height: 64px;
+		background: #EB6851;
+		text-transform: uppercase;
+		border-radius: 30px;
+	}
+}
+
+@media only screen and (max-width: 343px){
+	#mobile-cta {
+		text-transform: none; /* Stops button breaking in half on tiny screens */
 	}
 }
 
 /* BUTTON AND LINK STYLES */
 
-a:not(.button) {
+/*a:not(.button) {
 	color: #EB6851;
 	text-transform: uppercase;
 	letter-spacing: 2px;
-}
+}*/
 
-a:not(.button):hover {
+/*a:not(.button):hover {
 	color: #172751;
 	cursor: pointer;
-}
+}*/
 
```

Comment:
ok cool! i did mean to ask you about the button and then simply forgot in my rush to get this deployed before leaving the office. i'll definitely give your suggestion a try :)